lua/kodachi/ui/composer.lua

```diff
@@ -46,16 +47,20 @@ local function configure_current_as_composer(state)
   vim.wo.winfixheight = true
 
   -- Handle submitting
-  vim.cmd [[ inoremap <buffer> <cr> <cmd>lua require'kodachi.ui.composer'.submit()<cr> ]]
-  vim.cmd [[ nnoremap <buffer> <cr> <cmd>lua require'kodachi.ui.composer'.submit()<cr> ]]
+  vim.cmd [[inoremap <buffer> <cr> <cmd>lua require'kodachi.ui.composer'.submit()<cr>]]
+  vim.cmd [[nnoremap <buffer> <cr> <cmd>lua require'kodachi.ui.composer'.submit()<cr>]]
 
   -- Support inserting newlines
-  vim.cmd [[ inoremap <buffer> <s-cr> <cr> ]]
-  vim.cmd [[ inoremap <buffer> <a-cr> <cr> ]]
+  vim.cmd [[inoremap <buffer> <s-cr> <cr>]]
```

Comment:
Note the space paddings read nicely, but actually cause a `<Space>` to be included at the end of some of these mappings—not ideal.